code/game/objects/items.dm

```diff
@@ -65,6 +65,8 @@
 	var/embedded_unsafe_removal_pain_multiplier = EMBEDDED_UNSAFE_REMOVAL_PAIN_MULTIPLIER //The coefficient of multiplication for the damage removing this without surgery causes (this*w_class)
 	var/embedded_unsafe_removal_time = EMBEDDED_UNSAFE_REMOVAL_TIME //A time in ticks, multiplied by the w_class.
 
+	var/compost_value = 0 //Defines how many units of compost an item is worth.

```

Comment:
@Aranclanos @Iamgoofball First off, the var can't go in grown since it also applies to seeds. Second, if the var were to be any lower down then the compost tank would have to typecheck, which is an even worse solution than this.